README.md

```diff
@@ -99,6 +99,15 @@ An example `finch.yaml` looks like this:
 cpus: 4
 # memory: the amount of memory to dedicate to the virtual machine. (required)
 memory: 4GiB
+# snapshotter: the snapshotter a user want to use as there default snapshotter
+# Supported Snapshotters List:
+# - soci https://github.com/awslabs/soci-snapshotter/tree/main
+# Once the option has been set the snapshotter will be installed on either finch vm init or finch vm start.
+# The snapshotter binary will be downloaded on the virtual machine and will be configured and ready for use.
+# To change your default snpahotter back to overlayfs, simply remove the snapshotter value from finch.yaml
+# To completely remove the snapshotter binary, shell into your VM and remove /usr/local/bin/{snapshotter binary}
+# and remove the snapshotter configuration in the containerd config file found at /etc/containerd/config.toml
+snapshotter: soci
```

Comment:
If contianerd supports multiple snapshotters being enabled at once, this should probably be a slice

Reply:
That makes sense. My only question with that is how would the user specify what they want their default snapshotter to be? Should that be a separate config option to set the default snapshotter?

Reply:
We could have the default be the first snapshotter listed in this configuration. And then clearly document that. Additionally, users can specify snapshotter at the CLI on a per-command basis e.g. `finch run --snapshotter=<x> ...` - which again could be documented.